source/1_engine/rex_engine/src/diagnostics/win/win_debug.cpp

```diff
@@ -20,4 +23,18 @@ void rex::win::exit_confirm(int32 exitCode)
   {
     exit_process(exitCode);
   }
+}
+
+rsl::medium_stack_string rex::win::report_win_error(HResult hr, [[maybe_unused]] rsl::string_view winFunc, [[maybe_unused]] const rsl::string_view file, [[maybe_unused]] const rsl::string_view function, [[maybe_unused]] card32 lineNr)
+{
+  const _com_error err(hr);
+  rsl::medium_stack_string error_message(err.ErrorMessage());
+  REX_ERROR(LogEngine, "Windows Error");
+  REX_ERROR(LogEngine, "Win API: {}", winFunc);
+  REX_ERROR(LogEngine, "File: {}", file);
+  REX_ERROR(LogEngine, "Function: {}", function);
+  REX_ERROR(LogEngine, "On line: {}", lineNr);
+  REX_ERROR(LogEngine, "Windows error: {}", error_message);
```

Comment:
I'm not really a fan of this mulltiline error reporting, I even think that spdlog provides file, function, and line in their pattern thingy. Can't we use that and add everything on 1 line.

Reply:
the line becomes very long, that's why I split it to multiple lines.
imo it reads better

Reply:
IMO it does not =/

Reply:
Make it a bit more coherent please ;-)